docs/ossm/cert-manager/README.md

```diff
@@ -99,129 +119,160 @@ You can install the cert-manager tool to manage the lifecycle of TLS certificate
-        --install \
-        --namespace cert-manager \
-        --wait \
-        --set "app.tls.rootCAFile=/var/run/secrets/istio-csr/ca.pem" \
-        --set "volumeMounts[0].name=root-ca" \
-        --set "volumeMounts[0].mountPath=/var/run/secrets/istio-csr" \
-        --set "volumes[0].name=root-ca" \
-        --set "volumes[0].secret.secretName=istio-root-ca" \
-        --set "app.istio.namespace=istio-system"
-    ```
+    - Add the jetstack charts to your local helm repo.
+
+      ```sh
+      helm repo add jetstack https://charts.jetstack.io --force-update
+      ```
+
+    - Install the istio-csr chart.
+      ```sh
+      helm upgrade cert-manager-istio-csr jetstack/cert-manager-istio-csr \
+          --install \
+          --namespace cert-manager \
+          --wait \
+          --set "app.tls.rootCAFile=/var/run/secrets/istio-csr/ca.pem" \
+          --set "volumeMounts[0].name=root-ca" \
+          --set "volumeMounts[0].mountPath=/var/run/secrets/istio-csr" \
+          --set "volumes[0].name=root-ca" \
+          --set "volumes[0].secret.secretName=istio-root-ca" \
+          --set "app.istio.namespace=istio-system"
+      ```
 
     `RevisionBased` strategy installation
 
-    For the `RevisionBased` strategy, you need to specify all the istio revisions to your [istio-csr deployment](https://github.com/cert-manager/istio-csr/tree/main/deploy/charts/istio-csr#appistiorevisions0--string). You can find the names of your `IstioRevision`s with this command:
+    For the `RevisionBased` strategy, you need to specify all the istio revisions to your [istio-csr deployment](https://github.com/cert-manager/istio-csr/tree/main/deploy/charts/istio-csr#appistiorevisions0--string).
+
+    - Add the jetstack charts to your local helm repo.
+
+      ```sh
+      helm repo add jetstack https://charts.jetstack.io --force-update
+      ```
+
+    - Install the istio-csr chart with your revision name. Revision names will be of the form `<istio-name><istio-version-with-dashes>` e.g. `default-v1-23-0`.
+      ```sh
+      helm upgrade cert-manager-istio-csr jetstack/cert-manager-istio-csr \
+          --install \
+          --namespace cert-manager \
+          --wait \
+          --set "app.tls.rootCAFile=/var/run/secrets/istio-csr/ca.pem" \
+          --set "volumeMounts[0].name=root-ca" \
+          --set "volumeMounts[0].mountPath=/var/run/secrets/istio-csr" \
+          --set "volumes[0].name=root-ca" \
+          --set "volumes[0].secret.secretName=istio-root-ca" \
+          --set "app.istio.namespace=istio-system" \
+          --set "app.istio.revisions={default-v1-23-0}"
+      ```
+
+5.  Install your `Istio` resource.
+
+    Here we are disabling Istio's built in CA server and instead pointing istiod to the istio-csr CA server which will issue certificates for both istiod and the mesh workloads. We also mount the istiod tls cert into the pod at a known location where it will be read.
```

Comment:
Question: is it ok to call istio-csr as CA server? I though it's just some agent re-sending CSRs to cert-manager?
"which will issue certificates" - is istio-csr actually issuing the certs or just delegating?

Reply:
From the istiod and the workload pods perspective that's basically what it is since istio-csr is the entity processing their CSRs. istio-csr is just deleting to cert-manager which is delegating to the configured `Issuer` so maybe it's not technically correct to call it a CA server because unlike the istiod, it isn't the authority signing the certs.

Reply:
Now I'm remembering... the content of this I pulled from the 2.x documentation on cert-manager/istio-csr: https://docs.openshift.com/container-platform/4.16/service_mesh/v2x/ossm-security.html#ossm-cert-manager-integration-istio_ossm-security

Reply:
I think it's fine to leave it as it is. Thank you for clarifying, I was just curious.

Reply:
I updated the language to make it more clear that istio-csr is not the CA server. It merely receives the CSRs and then delegates to cert-manager to get the certificates signed.